main.tf

```diff
@@ -62,131 +62,132 @@ data "archive_file" "ami_cleanup" {
 
 module "label" {
   source    = "git::https://github.com/cloudposse/terraform-null-label.git?ref=tags/0.3.7"
```

Comment:
I guess this dependency is super out-dated and gives deprecation warnings. I see latest release there is `0.16.0`.
Probably everything would run clear when update it.